antarest/study/web/studies_blueprint.py

```diff
@@ -815,6 +816,26 @@ def unarchive_output(
         )
         return content
 
+    @bp.get(
+        "/private/studies/{study_id}/outputs/{output_id}/digest",
```

Comment:
If we consider it's really a UI-related endpoint (with already formatted tables instead of business related objects), we should probably make it clear in the path, like  "/private/studies/{study_id}/outputs/{output_id}/digest-ui"